Specs/DateTime/English/DateTimeModel.json

```diff
@@ -19104,13 +19104,86 @@
     "Context": {
       "ReferenceDateTime": "2011-07-02T00:00:00"
     },
-    "Results": []
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "week",
```

Comment:
This should either return "first week" and be un-resolved, or it should still return [].

Reply:
this is not clear: as for the issue description, I thought that only the unit "week" had to be extracted from "first week". Should we instead make it extract also the ordinal? (Returning [] is the previous behavior, so the second solution seems to imply to discard the whole PR).